difftools.md

```diff
@@ -49,6 +49,7 @@ This is how you can configure Git to open Meld when doing `git difftool`:
 
 ```shell
 $ git config --global diff.tool meld
+$ git config --global mergetool.meld.path "C:\Program Files (x86)\Meld\Meld.exe"
```

Comment:
using the above settings does not make meld to work from the command line: 
`diff.tool=meld
mergetool.meld.path=C:\Program Files (x86)\Meld\Meld.exe`

However, when running from the command line with full path "C:\Program Files (x86)\Meld\Meld.exe" does open the meld program.

Reply:
Nevertheless, the git difftool does work and open meld
`git difftool --no-index file1.txt file2.txt`

Reply:
I have got lost a bit in things I tried, but I think the solution requires to add 'C:\Program Files (x86)\Meld' to PATH environmental variable as well.

Reply:
Yes